app/models/subscriber.rb

```diff
@@ -2,4 +2,7 @@ class Subscriber < ActiveRecord::Base
   validates :address, presence: true
   validates :address, format: { with: /@/, message: "is not an email address" }
   validates :address, uniqueness: true
+
+  has_many :subscriptions, dependent: :destroy
+  has_many :subscription_lists, through: :subscriptions
```

Comment:
`subscriber_lists` here

Reply:
Clearly struggling with words this morning...